qucs-core/src/components/verilog/qucsMODULEdefs.xml

```diff
@@ -167,9 +167,46 @@
 <admst:variable name="nbr_nodes" select="%s"/>
 </admst:for-each>
 
-<admst:open file="$module.defs.h">
+<!-- output filename is 1st argument in -o flag in command line-->
+<admst:variable name="filename" select="%(/module/name)_common"/>
+<admst:variable name="filename_state" select="0"/>
+<admst:for-each select="/argv">
+  <admst:variable name="arg" select="%(.)"/>
+
+  <admst:if test="[position(.)=2]">
+    <admst:variable name="filename" select="$arg"/>
+  </admst:if>
+  <admst:if test="[$(filename_state)='1']">
+    <admst:variable name="filename" select="$arg"/>
+    <admst:variable name="filename_state" select="2"/>
+  </admst:if>
+  <admst:if test="[$(filename_state)='0' and $(arg)='-o']">
+    <admst:variable name="filename_state" select="1"/>
+  </admst:if>
+</admst:for-each>
+
+<admst:open file="$(filename).cpp">/*
```

Comment:
The idea here is just to accommodate the namespace or to pull all the model code inside a single file?